packages/fluent-theme/src/components/sendbox/index.tsx

```diff
@@ -119,24 +119,25 @@ function SendBox(
     <form {...aria} className={cx(classNames['sendbox'], props.className)} onSubmit={handleFormSubmit}>
       <SuggestedActions />
       <div className={cx(classNames['sendbox__sendbox'])} onClickCapture={handleSendBoxClick}>
-        <TelephoneKeypadSurrogate
-          autoFocus={true}
-          isHorizontal={false}
-          onButtonClick={handleTelephoneKeypadButtonClick}
-        />
         <TextArea
           aria-label={isMessageLengthExceeded ? localize('TEXT_INPUT_LENGTH_EXCEEDED_ALT') : localize('TEXT_INPUT_ALT')}
-          className={classNames['sendbox__sendbox-text']}
+          className={cx(classNames['sendbox__sendbox-text'], classNames['sendbox__text-area--in-grid'])}
```

Comment:
If we always apply `--in-grid`, why don't we merge both class together?

Reply:
You mean just removing the `--in-grid` prefix?
E.g.:
```
  > .sendbox__attachment {
    grid-area: attachment;
  }

  > .sendbox__sendbox-controls {
    grid-area: controls;
  }

  > .sendbox__telephone-keypad {
    grid-area: telephone-keypad;
  }
```

According to BEM this is essentially a modifier for the components to enable grid, so it uses the modifier postfix